openquake/mbt/tools/fault_modeler/fault_modeling_utils.py

```diff
@@ -25,7 +25,6 @@
 import numpy as np
 from copy import deepcopy
 import importlib
-
```

Comment:
Not sure if you deleted this on purpose, but it is a common practice to have imports in Python grouped into 2 or 3 groups, separated by blank lines:  
1. imports from the standard library
2. imports from external libraries (numpy should be in this instead of the top group)
3. imports from local libraries or those that are part of the project.

Reply:
okay, I fixed the grouping.